app/representers/space_representer.rb

```diff
@@ -27,5 +27,11 @@ class SpaceRepresenter < Roar::Decorator
     property :admin_id
     property :name
     property :picture
+
+    property :plan_details,
```

Comment:
It seems odd that this is going in the space representer... Why isn't this in the organization representer? Why does every space need to know details about its owning organizations plan?